app/views/account/index.html

```diff
@@ -8,50 +8,35 @@
 
 {% block formContent %}
 
-  <h1 class="govuk-heading-l">Create an account or sign in</h1>
-
-  {% set signinHtml %}
-    {{ govukInput({
-      label: {
-        text: "Email address",
-        classes: "govuk-label--s"
-      },
-      hint: {
-        text: "Enter the email address you used to register, and we’ll send you a link to sign in."
-      },
-      classes: "govuk-!-width-two-thirds",
-      attributes: {
-        spellcheck: "false"
-      },
-      autocomplete: "email",
-      type: "email"
-    } | decorateAttributes(data, "data.accountEmail")) }}
-  {% endset %}
+  <h1 class="govuk-heading-l">Create a GOV.UK account or sign in</h1>
+
+  <p class="govuk-body">You'll need:</p>
 
-  {{ govukRadios({
-    fieldset: {
-      legend: {
-        text: "Do you already have an account?",
-        classes: "govuk-fieldset__legend--m"
-      }
-    },
-    items: [
-      {
-        value: "Yes, sign in",
-        text: "Yes, sign in",
-        conditional: {
-          html: signinHtml
-        }
-      },
-      {
-        value: "No, I need to create an account",
-        text: "No, I need to create an account"
-      }
-    ]
-  } | decorateAttributes(data, "data.existingAccount")) }}
+  <ul class="govuk-list govuk-list--bullet">
+    <li>
+      an email address
+    </li>
+    <li>
+      a way to get security codes - this can be a UK mobile phone number or an authenticator app
+    </li>
+    
+  </ul>
 
   {{ govukButton({
-    text: "Continue"
+    text: "Create a GOV.UK account"
+  }) }}
+
+  <p class="govuk-body">If you already have a GOV.UK account you can <a href="./sign-in" class="govuk-link">sign in</a>.</p>
```

Comment:
This anchor should go to a page that looks like this:
https://ova-alpha.london.cloudapps.digital/govuk_account_sign_in